### PR TITLE
Security fix

### PR DESCRIPTION
**Pull request by santidrj**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/securingservice/FirebaseFactory.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5220023&bid=17115288)

**Review by AlbertPG — APPROVED** [review]
I see no problem at all

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5220023&bid=17115288)

File: src/test/java/org/pesmypetcare/webservice/controller/appmanager/MyPetCareRestControllerTest.java [hunk 1]
```diff
@@ -101,10 +103,13 @@ public void shouldNotPermitSignUpWithAnInvalidEmail() throws Exception {
...
             "The email is invalid", msg);
+        deleteUser1 = true;
     }
 
-    private static void deleteUser(String user) throws FirebaseAuthException {
-        FirebaseFactory.getInstance().getFirebaseAuth().deleteUser(user);
+    private static void deleteUser(String user) {
+        try {
+            FirebaseFactory.getInstance().getFirebaseAuth().deleteUser(user);
+        } catch (FirebaseAuthException ignored) { }
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/controller/appmanager/MyPetCareRestControllerTest.java]: Codacy Issue found: [Empty catch block.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5220023)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/securingservice/FirebaseFactory.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5220023&bid=17115288)

**Review by enrichp — APPROVED** [review]
;--------)

**Pull request merged**